applications/FluidDynamicsApplication/python_scripts/fluid_dynamics_analysis.py

```diff
@@ -71,7 +71,8 @@ def _GetOrderOfProcessesInitialization(self):
         return ["gravity",
                 "initial_conditions_process_list",
                 "boundary_conditions_process_list",
-                "auxiliar_process_list"]
+                "auxiliar_process_list",
+                "mesh_adaptivity_processes"]
```

Comment:
This is from FD app ... is this change necessary here ?

Reply:
no